src/stories/components/UserGreeting/UserGreeting.js

```diff
@@ -0,0 +1,54 @@
+import './styles.scss';
+
+import React, { useState, useEffect, useContext } from 'react';
+import UserServices from '../../../firebase/CRUDServices/userServices';
+
+export const UserContext = React.createContext();
```

Comment:
I wonder if we could get rid of this and just get the context value directly in line 18. I think that should work as long as the name of the context (in this case `UserContext`) is the same as the context created higher in the tree hierarchy. I saw that @cmutuc25 suggested moving the `createContext()` up to the App level component, and I agree with that. Maybe that could be done in this PR or maybe I could put up a quick separate PR to do that. Regardless, I think we could remove this and should be able to get the context value below.

Reply:
I agree with you. I saw that @cmutuc25 suggested to move the UserContext to the higher tree hierarchy, but I think maybe I still need to include this in this PR for now.